packages/components/modules/messages/EditGroup/index.tsx

```
@@ -67,6 +111,7 @@ const EditGroup: FC<EditGroupProps & { profileId: string }> = ({
       }
       delete dirtyValues.image
     }
+    delete dirtyValues.participants
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)